pkg/sidecar/fanout/fanout_handler.go

```diff
@@ -120,5 +121,11 @@ func (f *Handler) dispatch(msg *provisioners.Message) error {
 // makeFanoutRequest sends the request to exactly one subscription. It handles both the `call` and
 // the `sink` portions of the subscription.
 func (f *Handler) makeFanoutRequest(m provisioners.Message, sub eventingduck.ChannelSubscriberSpec) error {
+	if f.config.AsyncHandler {
+		f.logger.Info("><SB> Exercising ASYNC path in Fanout Request for", zap.Any("m = ", m), zap.Any("sub = ", sub))
```

Comment:
Probably shouldn't be committed. If is committed, should be Debug level.

Also, `zap.Any("m", m)` is sufficient (as compared to "m = "), because the string is used as a key in the printed JSON log.

Reply:
will be removed completely after debugging is concluded